guides/v2.1/cloud/docker/docker-config.md

```diff
@@ -30,16 +41,16 @@ This version also provides a ` docker:config:convert` command to convert PHP con
     ```bash
     composer install
     ```
-    
-    {: .bs-callout .bs-callout-info}
-    You can use the `--ignore-platform-reqs` option to bypass restrictions related to the PHP version.
 
 1.  In your local environment, start the Docker configuration generator.
 
     ```bash
-    vendor/bin/ece-tools docker:build
+    ./vendor/bin/ece-tools docker:build
     ```
 
+    {: .bs-callout .bs-callout-info}
+    You can use the `--php` option to specify the version of PHP compatible with your {{site.data.var.ee}} version.
```

Comment:
I think this note is fine, but I think you would only need to do this if you have more than one version of PHP installed, yes?   --   See the info on this page: https://devdocs.magento.com/guides/v2.2/install-gde/trouble/readiness/tshoot_rc_php.html

Reply:
For edification, this allows you to select the PHP image, so it is not dependent on local version.